### ISSUE TITLE
creates a yyyy.mm.dd directory

### ISSUE THREAD

**Issue by jorhett** [tagomoris/fluent-plugin-file-alternative#21]

**Comment by tagomoris** [comment]
It depends on file buffer plugin's behavior. We can fix it if needed, but its priority is low for me (any serious problems?)
Patches are welcome to fix it.

**Comment by jorhett** [comment]
Agreed that it's an annoyance not a problem, but we keep getting reports from users that it's a mistake on our part (since it's in a directory of directories named by date) and every once in awhile someone removes the directory and it stops the logging.
